public/createtopic.php

```diff
@@ -35,15 +29,13 @@
 $thisCategoryID = $forumData['CategoryID'];
 $thisCategoryName = $forumData['CategoryName'];
 
-getCookie($user, $cookieRaw);
```

Comment:
This was just fetching the user's cookie to pass the the API pages. See previous comment about my belief that functionality is unnecessary.